code/modules/client/preferences.dm

```diff
@@ -529,6 +531,10 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			if(jobban_isbanned(user, rank))
 				HTML += "<font color=red>[rank]</font></td><td><a href='?_src_=prefs;jobbancheck=[rank]'> BANNED</a></td></tr>"
 				continue
+			var/available_in_playtime = job.available_in_playtime(user.client)
+			if(available_in_playtime)
+				HTML += "<font color=red>[rank]<//font></td><td><font color=red> \[ " + get_exp_format(available_in_playtime) + " as " + job.get_exp_req_type()  + "\] </font></td></tr>"

```

Comment:
`+` triggers intermediate strings in the string table but `[]` and `text()` do not